src/libsyncengine/update_detection/update_detector/updatetree.cpp

```diff
@@ -93,7 +93,7 @@ std::shared_ptr<Node> UpdateTree::getNodeByPath(const SyncPath &path) {
     for (std::vector<SyncName>::reverse_iterator nameIt = names.rbegin(); nameIt != names.rend(); ++nameIt) {
         std::shared_ptr<Node> tmpChildNode = nullptr;
         for (const auto &childNode : tmpNode->children()) {
-            if (*nameIt == childNode.second->name()) {
+            if (Utility::isEqualNormalized(*nameIt, childNode.second->name())) {
```

Comment:
If we are on the same replica side, is it necessary to normalize before comparing file names?

Reply:
We are not necessary on the same replica. Especially, the method `OperationProcessor::findCorrespondingNodeFromPath` takes the path from one tree and use it to retrieve a node in the other tree.